### ISSUE TITLE
Allow setup without shared folders/files

### ISSUE THREAD

**Issue by coreyworrell** [mina-deploy/mina#537]
When setting shared folders/files to an empty array, or not setting the variables at all, `mina setup` will fail with the following:

```
$ mina setup
-----> Setting up /some/directory/
       bash: -c: line 4: syntax error near unexpected token `&&'
       bash: -c: line 4: `(cd /some/directory/shared &&  && cd -)'
       Connection to hostname.tld closed.

 !     Run Error
```

**Comment by ayghor** [comment]
+1

**Comment by coreyworrell** [comment]
Reference to pull request: #548

**Comment by lovro-bikic** [comment]
This bug was fixed with https://github.com/mina-deploy/mina/pull/592, which was released in [v1.2.3](https://rubygems.org/gems/mina/versions/1.2.3).
